docs/operations/using-rolling-upgrades.md

```diff
@@ -0,0 +1,44 @@
+---
+linkTitle: Platform Upgrade
+---
+
+# Overview
+
+This guide will demonstrate how to enable the orchestrator's rolling upgrades using the [Upgrade Guard](/docs/concepts/rolling-upgrades/#upgrade-guard) and [Node Manager](/docs/concepts/rolling-upgrades/#node-manager). This feature helps to prevent your persistent storage volumes from becoming unhealthy during an orchestrator update.
+
+# Prerequisites
+
+> ⚠️ Make sure you have met the requirements of [configuring a Pod Disruption Budget (PDB)](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
```

Comment:
I have folloed the metioned docs, but haven't found the `met the requirements`. There are two of them:
- `Your Kubernetes server must be at or later than version v1.21.` - Which is true, but we support 1.19+ now with beta PDB, so I can imagine this will clueless customers, who don't understand this.
- `You should confirm with your cluster owner or service provider that they respect Pod Disruption Budgets.` - The only important thing on the page. Because they don't have to understand how PDB works, it is our job, i'm not sure it is good ide to suggest them to follow the entire docs.

Reply:
@aeroniero33 please make sure to test if 1.19+ with beta PDB would work with the feature, thank you !